fuse_constraints/test/test_absolute_constraint.cpp

```diff
@@ -283,6 +283,57 @@ TEST(AbsoluteConstraint, Optimization)
   }
 }
 
+TEST(AbsoluteConstraint, PartialOptimization)
+{
+  // Create a variable
+  auto var = fuse_variables::Position3DStamped::make_shared(ros::Time(1, 0), fuse_core::uuid::generate("t1000"));
+  var->x() = 10.7;
+  var->y() = -3.2;
+  var->z() = 0.9;
+
+  // Create a partial constraint for the first and third indices
+  fuse_core::Vector2d mean1;
+  mean1 << 1.0, 3.0;
+  fuse_core::Matrix2d cov1;
+  cov1 << 1.0, 0.0,
+          0.0, 1.0;
+  std::vector<size_t> indices1 = {2, 0};
```

Comment:
Do you want to use the enum names here?
```suggestion
  std::vector<size_t> indices1 = {fuse_variables::Position3DStamped::Z, fuse_variables::Position3DStamped::X};
```